portingdb/tables.py

```diff
@@ -267,6 +267,27 @@ def __repr__(self):
                                        self.collection_package.name)
 
 
+class TrackingBug(TableBase):
+    u"""Tracking bugs associated with a package."""
+    __tablename__ = 'tracking_bugs'
+    __table_args__ = {'sqlite_autoincrement': True}
+
+    id = IDColumn()
+    collection_package_id = Column(
+        ForeignKey(CollectionPackage.id), nullable=False)
+    tracking_bug = Column(
+        Integer(), nullable=False,
+        doc='ID number of the tracking bug')
+
+    collection_package = relationship(
+        'CollectionPackage', backref=backref('tracking_bugs'))
```

Comment:
Look at at the bottom of tables.py for a few examples of doing an N-to-M relationship with intermediate table.